### PR TITLE
[book] [validation] Constraints can be applied to an entire class too

### PR DESCRIPTION
**Pull request by sustmi**

### CODE REVIEW

File: book/validation.rst [hunk 1]
```diff
@@ -511,8 +511,8 @@ For information on translating the constraint messages, see
 Constraint Targets
 ------------------
 
-Constraints can be applied to a class property (e.g. ``name``) or a public
-getter method (e.g. ``getFullName``). The first is the most common and easy
+Constraints can be applied to a class property (e.g. ``name``), a public
+getter method (e.g. ``getFullName``) or an entire class. The first is the most common and easy
```
Comment (xabbuh) [inline book/validation.rst]: The following sentence now doesn't match properly anymore. Can you reword that a bit?

Comment (sustmi) [inline book/validation.rst]: Can you be more specific @xabbuh ? What is the sentence not matching properly?

Comment (xabbuh) [inline book/validation.rst]: In the following sentence we mention "the first" and "the second" which is not exact anymore as we now have three things (property, method, and class).

**Comment by sustmi** [comment]
I updated the commit to mention all three validation targets.

File: book/validation.rst [hunk 2]
```diff
@@ -511,9 +511,11 @@ For information on translating the constraint messages, see
 Constraint Targets
 ------------------
 
-Constraints can be applied to a class property (e.g. ``name``) or a public
-getter method (e.g. ``getFullName``). The first is the most common and easy
-to use, but the second allows you to specify more complex validation rules.
+Constraints can be applied to a class property (e.g. ``name``), a public
+getter method (e.g. ``getFullName``) or an entire class. Property constraints
+are the most common and easy to use. Getter constraints allows you to specify
```
Comment (xabbuh) [inline book/validation.rst]: [...] constraints allow you [...]

**Comment by xabbuh** [comment]
:+1: This reads good to me. I just left one minor comment, but we can also fix that while merging.

File: book/validation.rst (hunk 2 above)
Comment (sustmi) [inline book/validation.rst]: Thanks. Fixed.

**Comment by sustmi** [comment]
The typo is fixed.

**Comment by WouterJ** [comment]
Thanks Miroslav, nice PR!

**Pull request merged**